e2e_tests/cli/convert/test_convert.py

```diff
@@ -0,0 +1,72 @@
+from os.path import dirname
+from pathlib import Path
+from pprint import pprint
+from typing import Tuple
+
+import pytest
+
+from e2e_tests.helpers import assert_cli, run_cli_command
+
+
+class TestYOLOv8SegmentedCli:
+    this_file_path = Path(dirname(__file__)).absolute()
+    data_path = (this_file_path / ".." / ".." / "data").resolve()
+
+    @pytest.fixture(autouse=True)
+    def config(self) -> None:
+        assert self.data_path.exists(), "Data path does not exist, tests cannot run"
+
+    def compare_directories(self, path: Path, expected_path: Path) -> Tuple[bool, str]:
+        """
+        Compare two directories recursively
+        """
+        assert path.exists() and expected_path.exists()
+        assert path.is_dir() and expected_path.is_dir()
+
+        for file in path.iterdir():
+            if file.is_dir():
+                # Recursively compare directories
+                result = self.compare_directories(file, expected_path / file.name)
+                if not result[0]:
+                    return result
+            else:
+                if file.name.startswith("."):
+                    # Ignore hidden files
+                    continue
+
+                # Compare files
+                with file.open("r") as f:
+                    content = f.read()
+
+                with Path(expected_path / file.name).open() as f:
+                    expected_content = f.read()
+
+                if content != expected_content:
+                    return (False, f"File {file} does not match expected file")
+
+        return (True, "")
+
+    @pytest.mark.parametrize(
```

Comment:
I'm a bit confused with this paramterization, is the intention that we'd extend this to other formats? In which case should the class be something more like TestConvertersCLI? or is the intention that the parameters would indicate different cases of yolov8?

Reply:
Yeah, that is the intention - see ticket for the full overview - but good call on the class name, I have just overlooked that rename.  Will action.